pkg/pgclient/config.go

```diff
@@ -75,6 +77,7 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	fs.IntVar(&cfg.WriteConnectionsPerProc, "db-writer-connection-concurrency", 4, "Maximum number of database connections for writing per go process.")
 	fs.IntVar(&cfg.MaxConnections, "db-connections-max", -1, "Maximum number of connections to the database that should be opened at once. It defaults to 80% of the maximum connections that the database can handle.")
 	fs.StringVar(&cfg.DbUri, "db-uri", defaultDBUri, "TimescaleDB/Vanilla Postgres DB URI. Example DB URI `postgres://postgres:password@localhost:5432/timescale?sslmode=require`")
+	fs.BoolVar(&cfg.PreferSimpleProtocol, "simple-protocol", defaultUseSimpleProtocol, "simple-protocol disables implicit prepared statement usage by the db driver. Set to true if using PGBouncer")
```

Comment:
Please update this in the `docs/cli.md`.

(feel free to ignore the suggestion)
```suggestion
	fs.BoolVar(&cfg.PreferSimpleProtocol, "simple-protocol", defaultUseSimpleProtocol, "Setting 'simple-protocol' to true disables the implicit prepared statement usage by the db driver. This can be done for using PGBouncer.")
```